src/main/resources/templates/dashboard.html

```diff
@@ -0,0 +1,222 @@
+{# A test page for the dual pane page #}
+{% extends "base/li-sidebar-page" %}
+{% block title %} Exception Null - This will override all other titles{% endblock %}
```

Comment:
For the final version, we should still set the titles for the various page views, though, even if it's as simple as setting it to "Exception Null - Dashboard" for this page